zebra-state/src/service/non_finalized_state.rs

```diff
@@ -701,10 +699,7 @@ impl NonFinalizedState {
                 // - the chain this bar was previously assigned to might have changed position.
                 chain_length_bar
                     .label(format!("Fork {fork_height}"))
-                    .set_pos(u64::try_from(chain.len()).expect("fits in u64"))
-                    .set_len(u64::from(
-                        zebra_chain::transparent::MIN_TRANSPARENT_COINBASE_MATURITY,
-                    ));
+                    .set_pos(u64::try_from(chain.len()).expect("fits in u64"));
```

Comment:
The chain length bar should either complete or get pruned within 100 blocks. If it doesn't, we should check if we're updating it before the block commit, rather than after. And we should check we're using the right constant, we might want MAX_BLOCK_REORG_HEIGHT instead.

Reply:
It panicked here after a while, I think it stops at 99? For precaution I kept it without length.